schema/crdb/internet-gateway/up08.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM omicron.public.router_route WHERE
+time_deleted IS NULL AND
+name = 'default-v6' AND
+vpc_router_id = '001de000-074c-4000-8000-000000000001';
```

Comment:
Migration notes:
* The IDs for the existing `inetgw:outbound` routes in fixed-data are `001de000-074c-4000-8000-000000000002` and `001de000-074c-4000-8000-000000000003`
* If we're deleting these entries we need to replace them with equivalents: the datastore route ensure will only manage the `Subnet` entries today.
* We need to create a default inetgw (and its link to the parent silo's default IP pool) per existing VPC.